src/PostgREST/Query/SqlFragment.hs

```diff
@@ -439,6 +442,12 @@ responseHeadersF = currentSettingF "response.headers"
 responseStatusF :: SQL.Snippet
 responseStatusF = currentSettingF "response.status"
 
+coalesceAddPgrstInsertedF :: SQL.Snippet
+coalesceAddPgrstInsertedF = "set_config('pgrst.inserted', (coalesce(" <> currentSettingF "pgrst.inserted" <> "::int, 0) + 1)::text, true) <> '0'"
+
+coalesceSubPgrstInsertedF :: SQL.Snippet
+coalesceSubPgrstInsertedF = "set_config('pgrst.inserted', (coalesce(" <> currentSettingF "pgrst.inserted" <> "::int, 0) - 1)::text, true) <> '-1'"
+
```

Comment:
This could be refactored to avoid repetition, something like:

```haskell
setConfigPgrstInserted add =
  let (sym, num) =  if add then ("+", "0") else ("-", "-1") in
  "set_config('pgrst.inserted', (coalesce(" <> currentSettingF "pgrst.inserted" <> "::int, 0) " <> sym <> " 1)::text, true) <> '" <> num <> "'"'"
```